src/ows/src/main/java/org/geoserver/ows/kvp/view/XMLViewParamsUtils.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.ows.kvp.view;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+public final class XMLViewParamsUtils {
+
+    private XMLViewParamsUtils() {}
+
+    /**
+     * Parses the XML string into a ViewParamsRoot entity bean.
+     *
+     * @param xmlStr the XML string to parse
+     * @return the parsed instance
+     */
+    public static ViewParamsRoot parseViewParams(String xmlStr) {
+        XmlMapper mapper = new XmlMapper();
```

Comment:
If it has any similarity to an ObjectMapper, it's expensive to create but also thread-safe, so it can be kept in a static field.

Reply:
Good point, applied.